pkg/app/mutate_app.go

```diff
@@ -155,3 +168,46 @@ func (m *Mutator) mutateKubeConfig(ctx context.Context, app v1alpha1.App) ([]mut
 
 	return result, nil
 }
+
+func (m *Mutator) mutateLabels(ctx context.Context, app v1alpha1.App) ([]mutator.PatchOperation, error) {
+	var result []mutator.PatchOperation
+
+	// Set app label if there is no app label present.
+	if key.AppKubernetesNameLabel(app) == "" && key.AppLabel(app) == "" {
+		result = append(result, mutator.PatchAdd(fmt.Sprintf("/metadata/labels/%s", replaceToEscape(label.AppKubernetesName)), key.AppName(app)))
```

Comment:
Setting the `app.kubernetes.io/name` label for `kgs validate apps`. @oponder Can you confirm this is the correct label?

Reply:
Looks correct to me!

Reply:
Thanks, it sets the label if there is no `app` or `app.kubernetes.io/name` for compatibility with existing app CRs managed by cluster-operator.